website/docs/r/group_profile.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "turbot"
+title: turbot
+template: Documentation
+page_title: "Turbot: turbot__group_profile"
+nav:
+  title: turbot__group_profile
+---
+
+# turbot_profile
+
+The `Turbot group profile` resource adds support for creating group profile under a directory type. It is used to create, manage and delete group profiles.
+
+## Example Usage
+
+**Creating Your First group profile**
+
+```hcl
+resource "turbot_group_profile" "test" {
+  title               = "terraform test group profile"
+  directory           = "112233445566"
+  group_profile_id    = "test"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `directory` - (Required) The parent directory of the group profile, either as an id, or an AKA.
+- `group_profile_id` - (Required)  The unique identifier of the group profile. For new group profiles this must be unique for the parent directory.
```

Comment:
Is it really called `group_profile_id`? It's not actually an ID :-( ... It's a name.

Might be nothing we can do now, but this is not consistent with anything else.....